config/local.yaml

```diff
@@ -0,0 +1,2 @@
+application:
+  host: 127.0.0.1
```

Comment:
Why is this different? Doesn't 0.0.0.0 work for this too?

Reply:
0.0.0.0 would work too. No strong reasoning other than it was an easy value to show for the local defaults.